bin/ubuntu-core-initramfs

```diff
@@ -21,13 +115,32 @@ def path_join_make_rel_paths(path, *paths):
 
 
 # Include kernel modules specified in conf_file
-def add_modules_from_file(dest_d, kernel_root, modules_d, fw_d, conf_file):
+def add_modules_from_file(dest_d, kernel_root, modules_d, fw_d, conf_file, db,
+                          allow_discoverable=True):
     with open(conf_file) as f:
         for module in f.readlines():
             module = module.strip()
             if module:
                 if module[0] in ["#", ";"]:
                     continue
+
+                try:
+                    mod = db[module]
+                except KeyError:
+                    print(f"NOTE: {conf_file}: Module {module} not found", file=sys.stderr)
+                else:
+                    if mod.builtin:
+                        print(f"NOTE: {conf_file}: Module {module} is builtin", file=sys.stderr)
+                    if not allow_discoverable and len(mod.aliases) > 0:
+                        print(f"WARNING: {conf_file}: Module {module} has aliases:", file=sys.stderr)
+                        for alias in mod.aliases:
+                            print(f" * {alias}", file=sys.stderr)
+                    # A module that has symbols but no alias is likely to be loaded through aliases
+                    if len(mod.aliases) == 0 and len(mod.symbols) > 0:
```

Comment:
I have pushed a commit adding this functionality. But it does not find more. `len(mod.aliases) == 0 and len(mod.symbols) > 0` is better because it also finds modules that are truly useless. If it is not installed by other modules, using dependencies would not find it. So I think it is redundant, and I am planning to remove that commit. Tell me what you think.